spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/jsbridge/BridgeCommand.java

```diff
@@ -46,9 +47,10 @@
     CONFIG_UPDATE_FORM(508),
     CONFIG_REGISTER_DEVICE(509),
     NOTIFICATIONS(600),
-    NOTIFICATION_ALERT(601),
-    NOTIFICATION_INFO(602),
-    NOTIFICATION_CONTENT_AVAILABLE(602);
+    //NOTIFICATION_ALERT(601),
```

Comment:
PR looks great!  really nick picky, commented out code seems like a waste of space :-), we could always use git history to see what was there before.